tests/rules/javascript/test.no_eval.js

```diff
@@ -0,0 +1,114 @@
+import { VALIDATION_WARNING } from 'const';
+import JavaScriptScanner from 'scanners/javascript';
+
+// These rules were mostly copied and adapted from
+// https://github.com/eslint/eslint/blob/master/tests/lib/rules/no-eval.js
+// Please make sure to keep them up-to-date and report upstream errors.
+describe('no_eval', () => {
+  var validCodes = [
+    'Eval(foo)',
```

Comment:
yeah, I removed those cases because we do have rules for it. Since our tests always test the whole test-suite at the moment (and I like to keep it that way tbh). We could separate this a bit but generally I like that we always look at the great picture for our rule tests.